src/engine/Input/Pointer.ts

```diff
@@ -309,6 +376,59 @@ export class Pointers extends Class {
       };
    }
 
+   private _handleWheelEvent(eventName: string, eventArr: WheelEvent[]) {
+      return (e: MouseWheelEvent) => {
+         // Should we prevent page scroll because of this event
+         if (this._engine.pageScrollPreventionMode === PageScrollPreventionMode.Page ||
+             (this._engine.pageScrollPreventionMode === PageScrollPreventionMode.Canvas && e.target === this._engine.canvas)) {
+            e.preventDefault();
+         }
+
+         var x: number = e.pageX - Util.getPosition(this._engine.canvas).x;
+         var y: number = e.pageY - Util.getPosition(this._engine.canvas).y;
+         var transformedPoint = this._engine.screenToWorldCoordinates(new Vector(x, y));
+
+         var deltaX = 0;
+         var deltaY = 0;
+         var deltaZ = 0;
+         var deltaMode = WheelDeltaMode.Pixel;
+
+         if (e.deltaX) {
+            deltaX = e.deltaX;
+         } else if (e.wheelDeltaX) {
+            deltaX = e.wheelDeltaX * (-1 / 40);
+         }
+
+         if (e.deltaY) {
```

Comment:
For everything being set to `deltaY` can we use a null coalesce trick to save some lines?

```typescript
deltaY = e.deltaY || (e.wheelDeltaY * CONSTANT) || (e.wheelDelta * CONSTANT) || e.detail
```